common/src/main/java/dev/schmarrn/lighty/ui/SettingsScreen.java

```diff
@@ -121,8 +127,8 @@ public void render(@NotNull GuiGraphics guiGraphics, int i, int j, float f) {
 
     @Override
     public void onClose() {
-        Compute.clear();
-        super.onClose();
+        assert this.minecraft != null;
```

Comment:
The `Compute.clear()` call is important to reset the overlay after changing the settings. It *must* be called when closing the settings menu.

```suggestion
        Compute.clear();
        assert this.minecraft != null;
```

Reply:
It's been restored.